### PR TITLE
New button and page created for affiliate projects

### PR DESCRIPTION
**Pull request by esloch**
--New button and page created for affiliate projects.

### CODE REVIEW

**Comment by esloch** [comment]
Screenshot_20190823_101612

File: output/pages/childrenprojects/index.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+<p>Specifics objectives:</p>
+<ul>
+<li>optimize data visualization;</li>
+<li>integration with open data repositories to publish data;</li>
+<li>reproducibility on data analysis tasks through storing and recovery operations;</li>
+</ul>
+<p>SkData should integrate with Pandas library (Python).</p>
+
+
+############################################################################################
```
Comment (xmnlab) [inline output/pages/childrenprojects/index.html]: alll these projects here could be removed. just keep skdata and ajna

**Review by xmnlab — CHANGES_REQUESTED** [review]
@esloch I just added some notes about something you should change.
about the name, change also the name of the folder from "childrenprojects" to "affiliated_projects"

File: output/pages/childrenprojects/index.html (hunk 1 above)
Comment (esloch) [reply output/pages/childrenprojects/index.html]: -  Skdata and Ajnaprojects will be kept!
- More info: Add Access Links to projects.
- Create a PopUp on the page on how to access the requirements to be an affiliate project.

**Review by esloch — COMMENTED** [review]
It will be done!
Thanks.

**Comment by xmnlab** [comment]
I think the `cache` folder shouldn't be stored into the repo.

**Comment by esloch** [comment]
-- I will provide a .gitignore; Any other files?

**Comment by xmnlab** [comment]
also ` __pycach__`

**Review by xmnlab — APPROVED** [review]

**Pull request merged**